xma/layers/linear_attention/module.py

```
@@ -82,22 +83,18 @@ def forward(
         key = key.view(*key.size()[:-1], self.num_key_heads, self.key_head_dim)
         value = value.view(*value.size()[:-1], self.num_value_heads, self.value_head_dim)
 
-        if input_state is not None:
-            input_state = input_state.view(-1, self.num_heads, self.key_head_dim, self.value_head_dim)
-
         input, input_state = linear_attention(
```

Ruff (0.16.1)

[error] Variable `input` is shadowing a Python builtin

(A001)